Token.js

```diff
@@ -848,27 +851,7 @@ class Token {
 				tok.find(".elev").show();
 			}
 
-			// HEALTH AURA / DEAD CROSS
```

Comment:
Same again on this one. it's called at the end of place via the 2 new functions

Reply:
The only thing that concerns me in this PR is that you're not taking this part 

    const pData = window.PLAYER_STATS[this.options.id] || this.options;

I'm not exactly sure what the difference is between `window.PLAYER_STATS` and `token.options` at this point in the code, but it looks like it's still used in other places so you should probably check it in your `update_dead_cross` and `update_health_aura` functions. If I'm wrong, and the token is already synchronized with these values, then using the token values is just fine. I'm not familiar with how player stats get sync'd, but I noticed that this one was slightly different from the others, and wnated to point it out just in case.